sdk/Sdk/Targets/Microsoft.Azure.Functions.Worker.Sdk.targets

```diff
@@ -66,6 +66,16 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
         <Error Condition="'$(_IsFunctionsSdkBuild)' == 'true'" Text="Microsoft.NET.Sdk.Functions package is meant to be used with in-proc function apps. Please remove the reference to this package in isolated function apps."/>
     </Target>
 
+    <PropertyGroup Condition="$(TargetFrameworkIdentifier) == '.NETFramework'">
+        <_FunctionsWorkerExecutableFileName >$(TargetName).exe</_FunctionsWorkerExecutableFileName>
```

Comment:
There is no single variable present in the `Microsoft.NET.Sdk` targerts file to get this value.

Reply:
You can use `$(TargetName)$(TargetExt)`

Reply:
Yea. Switched to the variable in the latest iteration. BTW, My comment was about the absence of the one variable which has the right executable file name value in all different scenarios.

Reply:
I meant you do not need to include any of your setting of `_FunctionsWorkerExecutableFileName`. `$(TargetName)$(TargetExt)` will work for all scenarios (.dll, .exe, win-runtime .exe, etc).